packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -25,12 +32,10 @@ class FramesBarPlotly {
   Layout getFPSTimeseriesLayout() {
     return Layout(
       xaxis: AxisLayout(
-        ticks: '',
-        showgrid: false,
-        showticklabels: false,
-        rangeslider: RangeSlider(
-          rangemode: 'nonnegative',
-          autorange: true,
+        rangeslider: RangeSlider(),
+        // Hide ticks by using font color of bgColor.
+        tickfont: Font(
+          color: 'white',
```

Comment:
Add TODO to make this color work with the dark theme.

Reply:
Done.